### ISSUE TITLE
Where to upload Header Image in Enterprise Theme

### ISSUE THREAD

**Issue by lisandi** [humhub/humhub-modules-enterprise-theme#23]
Where can you actually upload the header image that gets displayed also as a nice header in the spaces Menu be uploaded in the Enterprise Theme?

I uploaded that image of my great grand son that gets displayed in AY Space while switching to the HumHub Theme and then switching back to the Enterprise Theme.

I would suggest to add the header image as a Header in the About Page of the Enterprise Theme instead - But I could not find the template actually for the About page either 😉.

Of course you could add the image as an Image but then it would not get displayed in the header of the spaces directory entry. Also the Image would be missing when sharing a space on social media.

https://community.humhub.com/content/perma?id=256357

**Comment by luke-** [comment]
Currently there are no space headers in the Enterprise theme. But this is planned.

**Comment by lisandi** [comment]
@luke- yes I know but it is not at all useful as when there is no way to define a header image you have also no background image for the directory entry header for each space. The only workaround right now is constantly to have to switch forth and back between the HumHub and the Enterprise Theme and that is really no good solution. So I would recommend to have at least a way in Enterprise Theme to define a Header Image - similar on how yo could define a Profile (Logo) Image.
The look of the Enterprise Theme is actually good as is but I would recommend making it possible that i.e. a starting page could be defined on which then the header image would be shown. That starting page could be also the about page. I tried to find actually the place where the about page gets defined but was not at all successful, perhaps you could help by telling me where to look for that about page. The idea is to simply adding an option to have the header image that appears in the HumHub Theme on top of every page also appearing on the about page as an option i.e. when no other image specific for that about page got defined. Currently it is only possible to upload any image as image on the about page but then the Title would be above that image and I like to get that Title to be moved below that image. So where is the place to look at? That could be a change that could be available for both Themes actually - in About on HumHub Theme as an option to insert a second header image or to take the already inserted one and on the Enterprise Theme it would work too. but then full page width.

**Comment by lisandi** [comment]
Customers complain that they can't actually upload a header image as they can easily in HumHub Theme and that is a BLOCKER to get new projects up and running. No one really understands why you have constantly to switch forth and back between HumHub Theme and Enterprise Theme ONLY to have finally also a Header image in the spaces directory visible instead of a great default background. I hope that can be added and released with the next update as it is really MISSING! You need that header image for the spaces directory header of each directory blubs. Thanks!
![Screen Shot 2022-07-22 at 16 18 56](https://user-images.githubusercontent.com/1982011/180408272-383d2365-3938-40e5-b203-1d66446b09a4.png)
Header Banner Image can easily be added and modified
![Screen Shot 2022-07-22 at 16 19 13](https://user-images.githubusercontent.com/1982011/180408299-312e1774-1108-40e0-a93c-491000d65a74.png)
Header Banner Image displays nicely as Header Banner Image of each Blub
![Screen Shot 2022-07-22 at 16 20 29](https://user-images.githubusercontent.com/1982011/180408303-5bf79e3e-f84f-4bf5-978a-f87c13bb0c8c.png)
A Header Banner Image does not even exist in the Enterprise template, i.e. it could be also the featured image for a space that shows up when sharing a space or the default image when sharing content from space on other social media i.e. FB/ Twitter, etc. But when Enterprise Theme active there is definitely no way to upload a Header Banner Image. The image in About Youtube - Vimeo has been changed by switching to HumHub Theme but the ASLM.DEMO Header image can't be modified or even added directly without switching from Enterprise Theme.
![Screen Shot 2022-07-22 at 16 20 53](https://user-images.githubusercontent.com/1982011/180408313-01202d6f-8ba3-49d9-bc93-2774ea62bcb6.png)
The Blub shows a Grey Header Banner Image because if can't be added when Enterprise Theme Active, even not from the ABOUT Settings Images Tab - Here you can only define a Logo for a space but not its header banner image.
![Screen Shot 2022-07-22 at 16 21 07](https://user-images.githubusercontent.com/1982011/180408321-e81f3085-f696-402f-9849-705afe5be4ed.png)
